src/connection-manager/index.js

```diff
@@ -178,7 +178,7 @@ class ConnectionManager extends EventEmitter {
     const total = received + sent
     this._checkMaxLimit('maxData', total)
     log('metrics update', total)
-    this._timer.reschedule(this._options.pollInterval)
+    retimer(this._checkMetrics, this._options.pollInterval)
```

Comment:
Need to set this, otherwise we can't clear on stop.
```suggestion
    this._timer = retimer(this._checkMetrics, this._options.pollInterval)
```